src/components/button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styles from "./style.module.scss"
+
+class Button extends React.Component{
+    render(){
+        return(

```

Comment:
Presta atención a los espaciados entre las instrucciones para que el código sea más legible.
```suggestion
        return (
```